brainstormforce/ultimate-addons-for-gutenberg
# Fix: Price List - Issue when imagePosition is undefined when used as a reusable block.

### Description

- https://trello.com/c/a1ETE3OZ/196-price-list-issue-when-used-as-a-reusable-block

### Screenshots


### Types of changes




- Bug fix

### How has this been tested?

- Create a price list as a reusable block in master ad images.
- Drag and drop on diff pages, you will see the issue.
- Switch to this branch to refresh the page where the price list reusable block is used. Issue resolved

### Checklist:
